torchvision/transforms/v2/_misc.py

```diff
@@ -415,7 +416,7 @@ def forward(self, *inputs: Any) -> Any:
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         is_label = inpt is not None and inpt is params["labels"]
-        is_bounding_boxes_or_mask = isinstance(inpt, (tv_tensors.BoundingBoxes, tv_tensors.Mask))
+        is_bounding_boxes_or_mask = isinstance(inpt, (tv_tensors.BoundingBoxes, tv_tensors.DetectionMasks))
```

Comment:
We definitely don't want to do that because we don't want to force users to use `DetectionMasks`. 

Both `DetectionMasks` and `SegmentationMask` must be 100% inter-changeable with just `Mask`. Maybe that won't be the case in the future, but right now it should